src/model/paths_types.ml

```diff
@@ -592,10 +597,16 @@ module rec Reference : sig
     | `TChildPage
     | `TChildModule ]
 
+  type page_path =
```

Comment:
nitpick on the name: I think it is a bit misleading, those kind of references can also be used to reference modules. From the [RFC](https://github.com/ocaml/odoc/discussions/1097):

> A module can be added to the scope by adding the library it's found in, which will make those modules accessible by the references `{!Module}` or **`{!/libname.sublib/Module}`** (not `{!sublib/Module}`). By default all modules in the package's libraries are added to the scope.

Reply:
Let me be constructive and suggest something! Maybe `hierarchical_path`?

Reply:
True. What do you think of `path` ?

Reply:
I am slightly reluctant (we already have `Paths`, `Paths.Path`, references also technically are paths, ...).

Let me propose `hierarchy`. I think it is a bit more explicit, and avoids restating a third time that it is a path.